onnx/external_data_helper.py

```diff
@@ -183,6 +183,21 @@ def convert_model_from_external_data(model: ModelProto) -> None:
             tensor.data_location = TensorProto.DEFAULT
 
 
+def _is_path_traversal(path: str) -> bool:
+    """Check if the path contains traversal sequences or is absolute.
+
+    Arguments:
+        path: The path to check.
+    """
+    # Check for absolute paths
+    if os.path.isabs(path):
+        return True
+    
+    # Check for path traversal patterns
+    parts = path.replace("\\", "/").split("/")
```

Comment:
Is this needed, or is just seeing `..` sufficient to tell us there's traversal going on?